download_cmd.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/facebookgo/stackerr"
+	"github.com/spf13/cobra"
+)
+
+type downloadCmd struct{}
+
+type releaseFiles struct {
+	UserFiles map[string]map[string]string `json:"userFiles"`
+	Checksums map[string]map[string]string `json:"checksums"`
+}
+
+type fileMap map[string][]byte
```

Comment:
do not like this storing stuff in memory.
perhaps create the full folder struct in a temp location & make a filesystem mv to current dir when required.